Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 public extension LogtoCore {
     static let postHeaders: [String: String] = [
-        "Content-Type": "application/x-www-form-urlencoded",
+        "Content-Type": "application/x-www-form-urlencoded; charset=UTF-8",
```

Comment:
Is this a necessary config on swift side?

Reply:
no, just for testing purpose.